bosh/secrets.example.yml

```diff
@@ -0,0 +1,11 @@
+# These are secrets because the volume names need to be secret,
+# or a customer will see this and be able to mount up another
+# customer's volumes.  These should probably be something like a uuid.
```

Comment:
How about dropping the "probably": "Volume names should be UUIDs."

Reply:
Good call.